pkg/ccl/workloadccl/cliccl/fixtures.go

```diff
@@ -127,7 +127,7 @@ const storageError = `failed to create google cloud client ` +
 // caller is responsible for closing it.
 func getStorage(ctx context.Context) (*storage.Client, error) {
 	// TODO(dan): Right now, we don't need all the complexity of
-	// storageccl.ExportStorage, but if we start supporting more than just GCS,
+	// storage.cloud.ExternalStorage, but if we start supporting more than just GCS,
```

Comment:
nit: in go code we usually just refer to the symbol as pkg.Name and leave the package path implicit i.e. no `storage.` prefix here.